src/clang-cpp-frontend/clang_cpp_convert.cpp

```diff
@@ -1085,17 +1085,27 @@ bool clang_cpp_convertert::get_function_body(
           {
             exprt lhs;
             // parsing non-static member initializer
-            if (get_decl_ref(*init->getMember(), lhs))
+            if (get_decl_ref(*init->getMember(), lhs, false))
```

Comment:
Maybe we can set an `irep_namet` to avoid adding a new variable? such as `lhs.set("#member_init", 1);`